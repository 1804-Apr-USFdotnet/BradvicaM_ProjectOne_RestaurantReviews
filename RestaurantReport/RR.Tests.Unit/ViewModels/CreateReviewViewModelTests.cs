using System;
using ApprovalTests;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using RR.ViewModels;

namespace RR.Tests.Unit.ViewModels
{
    [TestClass]
    public class CreateReviewViewModelTests
    {
        [TestMethod]
        [UseReporter(typeof(DiffReporter))]
        public void ToString_OnCall_ReturnsCorrectInformation()
        {
            var viewModel = new CreateReviewViewModel
            {
                Comment = "Some Comment",
                Rating = 1.0,
                ReviewerName = "Some Reviewer",
                RestaurantPublicId = Guid.Empty
            };

            Approvals.Verify(viewModel);
        }
    }
}
