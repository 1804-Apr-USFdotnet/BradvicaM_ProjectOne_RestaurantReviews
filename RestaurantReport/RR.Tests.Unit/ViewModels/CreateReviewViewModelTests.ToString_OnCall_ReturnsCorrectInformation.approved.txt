
RestaurantPublicId: 00000000-0000-0000-0000-000000000000
ReviewerName: Some Reviewer
Rating: 1
Comment: Some Comment
